Gemfile

```diff
@@ -17,15 +17,21 @@ gem 'pronto-sorbet', require: false
 
 # Rubocop Extentions
 # see https://docs.rubocop.org/rubocop/extensions.html
-gem 'rubocop-performance', require: false
-gem 'rubocop-rails', require: false
-gem 'rubocop-rspec', require: false
-gem 'rubocop-thread_safety', require: false
-gem 'rubocop-require_tools', require: false
-gem 'rubocop-i18n', require: false
-gem 'rubocop-sequel', require: false
-# gem 'cookstyle', require: false # don't support latest rubocop
-gem 'rubocop-rake', require: false
-
-gem 'rubocop-md', require: false
-gem 'rubocop-minitest', require: false
+## Rubocop Official Extentions
+gem 'rubocop-performance', '~> 1', require: false
+gem 'rubocop-rails', '~> 2', require: false
+gem 'rubocop-rspec', '~> 2', require: false
+gem 'rubocop-minitest', '~> 0', require: false
+gem 'rubocop-rake', '~> 0', require: false
+gem 'rubocop-sequel', '~> 0', require: false
+gem 'rubocop-thread_safety', '~> 0', require: false
+## Third-party Extensions
+gem 'rubocop-require_tools', '~> 0', require: false
+gem 'rubocop-i18n', '~> 3', require: false
+# gem 'cookstyle', '~> 7', require: false
+gem 'rubocop-packaging', '~> 0', require: false
+gem 'rubocop-sorbet', '~> 0', require: false
+gem "unparser", '> 0', require: false # for Sorbet/SignatureBuildOrder cop
+gem 'rubocop-graphql', '~> 0', require: false
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `rubocop-graphql` should appear before `unparser`.